framework/test/bench/AnyMapPerfTest.cpp

```diff
@@ -0,0 +1,190 @@
+#include "benchmark/benchmark.h"
+
+#include <cppmicroservices/Bundle.h>
+#include <cppmicroservices/AnyMap.h>
+#include <cppmicroservices/BundleEvent.h>
+#include <cppmicroservices/BundleContext.h>
+#include <cppmicroservices/Framework.h>
+#include <cppmicroservices/FrameworkFactory.h>
+#include <cppmicroservices/FrameworkEvent.h>
+
+#include <iostream>
+#include <cassert>
+
+#include "TestUtils.h"
+
+using namespace cppmicroservices;
+
+class AnyMapPerfTestFixture : public ::benchmark::Fixture
+{
+public:
+  using benchmark::Fixture::SetUp;
+  using benchmark::Fixture::TearDown;
+
+  void SetUp(const ::benchmark::State&)
+  {
+    using namespace cppmicroservices;
```

Comment:
the cppmicroservices namespace was already brought in on line 16

Reply:
Done.